src/Playwright/Program.cs

```diff
@@ -39,6 +39,15 @@ public static int Main(string[] args)
 
         public int Run(string[] args)
         {
+            // we need to use this logic, because .NET by design does some weird magic with the quotes
+            // so, we'll use a bit of our own magic... We'll start by calling the same base mathod, however
+            // we know it'll return the executable & path in the first argument. We'll use this to replace
+            // the entire string, which we can access from the "original" command line that we were called with
+            // which doesn't have the quotes stripped, but does include the full path
```

Comment:
I'd abbreviate the comment to something like 'Use original command line which doesn't have the quotes stripped.'